extension/json/json_functions/json_create.cpp

```diff
@@ -207,6 +207,9 @@ struct CreateJSONValue<INPUT_TYPE, int64_t> {
 template <class INPUT_TYPE>
 struct CreateJSONValue<INPUT_TYPE, double> {
 	static inline yyjson_mut_val *Operation(yyjson_mut_doc *doc, const INPUT_TYPE &input) {
+		if (Value::IsNan(input) || !std::isfinite(input)) {
```

Comment:
Can you change this to `Value::IsFinite`? This checks for both `inf` and `nan` at the same time.